<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormChild</class>
 <widget class="QWidget" name="FormChild">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Battery" name="widget_Battery" native="true">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>241</width>
     <height>261</height>
    </rect>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
   <widget class="QLabel" name="label_Battery">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>67</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Battery_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>水平</string>
    </property>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>271</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Widget
{	
	background-color: rgb(85, 141, 127);
}
*
{
	font:30pt;
}
#WidgetShowImage
{
	border:2px solid black;
}

#label
{
	background-color: rgb(85, 141, 127);
	color: rgb(255, 255, 255);
	border-image: url(:/images/transall.png);
	font: 28pt;
	text-align : left;
}
QPushButton
{
        background: rgb(68, 69, 73);
        color: white;
	/*boder-top:2px solid rgb(255,0,183);*/
}
QPushButton:!checked
{
        background: rgb(255, 0, 0);
        color: rgb(200, 200, 200);
}

#tableWidget
{
	background-color:rgba(117,117,117);
}
#tableWidget:item
{
	/*border-top:1px solid rgb(181,182,183);
	border-bottom:1px solid rgb(71,68,67);*/
	margin-left:40px;
	margin-right:40px;
}/*
#scrollArea,#scrollAreaWidgetContents
{	
	border:none;
	background-color:rgba(117,117,117,0);
}*/
#btn_2
{
	background-color: rgb(85, 255, 255);
        color: white;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>旋转====</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>50</y>
     <width>96</width>
     <height>68</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>旋转</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>430</y>
     <width>221</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>240</y>
     <width>190</width>
     <height>190</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>220</y>
     <width>190</width>
     <height>190</height>
    </rect>
   </property>
   <property name="text">
    <string>按钮设置背景图</string>
   </property>
  </widget>
  <widget class="BatteryNew" name="widget_Battery_New" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>211</width>
     <height>261</height>
    </rect>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
   <widget class="QLabel" name="label_Battery_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>67</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Battery_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>色带</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>490</y>
     <width>97</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>电池电量</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_img" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>451</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BatteryNew</class>
   <extends>QWidget</extends>
   <header>batterynew.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Battery</class>
   <extends>QWidget</extends>
   <header>battery.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
